### ISSUE TITLE
RuntimeException with multiple connections

### ISSUE THREAD

**Issue by vbartusevicius** [Orbitale/CmsBundle#7]
in `Resources/config/services.yml:20` is hardcoded  `connection: default`.
This fails when doctrine uses multiple connections:

```
doctrine:
    dbal:
        connections:
            host_connection:
                # params
            sock_connection:
                # params
    orm:
        connection: %default_connection%
```

when run `app/console cache:clear` I got:

```
[RuntimeException]                                                                                                                                                    
The Doctrine connection "default" referenced in service "gedmo.listener.timestampable" does not exist. Available connections names: host_connection, sock_connection 
```

There should be some way to avoid a hardcoded connection. Maybe take it from ORM or from `doctrine.dbal.default_connection` parameter.

**Comment by Pierstoval** [comment]
I'm thinking about refactoring this and remove the entity behaviors to let the user implement his own.

As for now for 1.x I might add default connection as configuration option :)

**Comment by vbartusevicius** [comment]
@Pierstoval - `connection` as bundle configuration would be a great and quick solution

**Comment by Pierstoval** [comment]
Available in 1.6.0, but not working, so fixed in 1.6.1 :wink:
